rosdep/base.yaml

```diff
@@ -2351,6 +2357,14 @@ libmysqlcppconn-dev:
   gentoo: [dev-db/mysql-connector-c++]
   opensuse: [libmysqlcppconn-devel]
   ubuntu: [libmysqlcppconn-dev]
+libncurses:
+  arch: [ncurses]
+  debian: [libncurses5]
+  fedora: [ncurses]
+  gentoo: [sys-libs/ncurses]
+  macports: [ncurses]
```

Comment:
@iluetkeb if you don't actually have any systems using this macports package I think you can leave it out as it would seem Homebrew is the ascendant package manager for Mac-based development these days.